site/content/en/guides/producer/init/_index.md

```diff
@@ -8,11 +8,11 @@ description: >
 ---
 
 A kpt package is published as a git subdirectory containing configuration
-files (YAML). Publishes of kpt packages can create or generate YAML files
+files (YAML). Publishers of kpt packages can create or generate YAML files
 however they like using the tool of their choice.
 
 Publishing a package is done by pushing the git directory
-(and optionally tagging it with a version).
+(and optionally [tagging](https://git-scm.com/book/en/v2/Git-Basics-Tagging) it with a version).
```

Comment:
Follow the convention of having all links at the bottom of the doc so they can be updated easily.

Reply:
Done